loki-v2-engine-dataobjects-research.md

```diff
@@ -0,0 +1,213 @@
+# Loki V2 Query Engine and Data Objects - Research Summary
```

Comment:
Do we want to have these docs at the root level of the repo?  Or do we want to maybe create a folder for loki/docs/internal ?

Reply:
no, this wasn't mean to be comitted